luzer/luzer_args.c

```diff
@@ -0,0 +1,284 @@
+/*
+ * SPDX-License-Identifier: ISC
+ *
+ * Copyright 2022-2023, Sergey Bronnikov
+ */
+
+#include <lauxlib.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "luzer_args.h"
+#include "macros.h"
+
+#define ENV_NOT_USE_CLI_ARGS "LUZER_NOT_USE_CLI_ARGS"
+
+#define SEED_CORPUS_PATH_FLAG "-corpus"
+
+#define FLAG_SCANF_FORMAT_KEY "-%[^=]"
+#define FLAG_PATTERN_KEY "-%s="
+#define FLAG_PATTERN_KEY_VALUE "-%s=%s"
+#define FLAG_PATTERN_OVERHEAD 2
+#define FLAG_FULL_SIZE(flag) (strlen(flag) + FLAG_PATTERN_OVERHEAD)
+
+/* Structure for convenient argument parsing. */
+typedef struct {
+	char **argv;
+	int argc;
+} luzer_args;
+
+NO_SANITIZE static bool
+is_flag_in_args(luzer_args *f_args, const char *key) {
+	if (!f_args || !f_args->argv || f_args->argc <= 1) {
+                return false;
```

Comment:
Why functions below returns `int` and here you are using `bool`?

Reply:
Do you mean to change return type to int? With return 0/1?